docs/testing_strategies.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Taquito Testing Strategies
+id: testing_strategies
+author: Michael Kernaghan
+---
+ 
+### How Taquito is Tested
+ 
+- General Philosophy
```

Comment:
could we try and use H4 instead here? 

e.g. 
'#### How Taquito is Tested'

Reply:
changed to 
#### General Philosophy